pkgs/applications/audio/airwindows/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv, lib, fetchFromGitHub, requireFile, unzip, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "airwindows";
+  version = "unstable-2021-12-24";
+
+  src = fetchFromGitHub {
+    owner = "airwindows";
+    repo = "airwindows";
+    rev = "0e28ead5b7dd759ad07ff7bfbd95e1e458e7fcd2";
+    sha256 = "sha256-QUM7dKPcan9g3miZFtaf4oblYVnJ86iJiyOKi70jV/E=";
+  };
+
+  vst-sdk = stdenv.mkDerivation rec {
+    name = "vstsdk366_27_06_2016_build_61";
+    src = requireFile {
+      name = "${name}.zip";
+      url =
+        "https://www.steinberg.net/sdk_downloads/vstsdk366_27_06_2016_build_61.zip";
+      sha256 = "05gsr13bpi2hhp34rvhllsvmn44rqvmjdpg9fsgfzgylfkz0kiki";
+    };
+    nativeBuildInputs = [ unzip ];
+    installPhase = "cp -r . $out";
+  };
+
+  airwindows-ports = stdenv.mkDerivation rec {
+    name = "airwindows-ports";
+    src = fetchFromGitHub {
+      owner = "ech2";
+      repo = "airwindows-ports";
+      rev = "0.4.0";
+      sha256 = "1ya4qbc63sb52nzisdapsydrnnpqnjsl5kgxibbr3dxf32474g89";
+    };
+    installPhase = "cp -r . $out";
+  };
```

Comment:
If we copy everything to $out and do nothing else we can also only use fetchFromGitHub.

Reply:
I don't understand.
I thought you meant to remove the installPhase, but that errors out.

Reply:
We could do something like https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/fira/default.nix